src/component/user/Authentication/Authentication.tsx

```diff
@@ -15,7 +15,10 @@ import ConditionallyRender from '../../common/ConditionallyRender';
 import { Alert } from '@material-ui/lab';
 import { useAuthDetails } from '../../../hooks/api/getters/useAuth/useAuthDetails';
 
-const Authentication = () => {
+interface IAuthenticationProps {
+    redirect: string;
+}
+const Authentication = ({ redirect }: IAuthenticationProps) => {
```

Comment:
We need to validate the redirect to avoid the unvalidated redirect vulnerability.

https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html

Reply:
@ivarconr I think that we are good as we are redirected to 404 page if we put a link not existing. Even if we try to redirect to external websites it won't work.
Also I did test it with subpath: https://www.loom.com/share/262a82afd4ad4e5b9581331bb15286f8 and it is working ✅

Reply:
perfect, thanks!